README.md

```diff
@@ -1,168 +1,297 @@
+- [IAccount and access to claims](https://docs.microsoft.com/en-us/azure/active-directory/develop/accounts-overview)
+- Enable Single Sign-On with different [authorization agents](https://docs.microsoft.com/en-us/azure/active-directory/develop/authorization-agents)
+- Support for synchronous methods from worker threads
+>NOTE: Tyler -  Think there may be some outdated config info on docs.ms under concepts->app scenarios->mobile app that calls web APIs. Where does new doc live?
 
-    // ...
+- Improved configuration and control of your PublicClientApplication using configuration file
+- AndroidX Compatible
 
-    // The access token can now be used to access a protected API!
-    String accessToken = authenticationResult.getAccessToken();
-```
+### Migrating from ADAL
+>NOTE: check this is right link
 
-For a full example, checkout the [code sample](https://github.com/Azure-Samples/active-directory-android-native-v2).
+You can review the ADAL to MSAL migration guide [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/migrate-android-adal-msal)
 
-## Installation
+### Migrating from preview versions of MSAL
 
-### Binaries via Gradle (Recommended way)
+>NOTE: where does this doc live?  
 
-Add to your app's build.gradle:
+You can review the MSAL Preview to GA release guide here
 
-```gradle
-    dependencies {
-        implementation 'com.microsoft.identity.client:msal:0.2.+'
-        }
-    }
-```
+### Sample
+>NOTE: shane- should we link kotlin sample?
 
-### AAR package inside libs folder
-You can get the AAR file from maven central and drop it into **libs** folder of your project.
+For a complete running sample [link to new sample](https://github.com/Azure-Samples/ms-identity-android-java)
 
-## Community Help and Support
-
-We use [StackOverflow](http://stackoverflow.com/questions/tagged/msal) with the community to provide support. You should browse existing issues to see if someone has asked about your issue before. If there are workable solutions to your issue then try out those solutions. If not, ask your question and let the community help you out. We're part of the community too and watch for new questions. We help with answers when the community cannot give you a solution.
-
-If you find and bug or have a feature request, please raise the issue on [GitHub Issues](../../issues). 
-
-## Contribute
+## Using MSAL
 
-We enthusiastically welcome contributions and feedback. You should clone the repo and start contributing now. 
+- Before you can get a token from Azure AD v2.0 or Azure AD B2C, you'll need to register an application. To register your app, use [the Azure portal](https://aka.ms/AppRegistrationsPreview). For Azure AD B2C, checkout [how to register your app with B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-app-registration).  
```

Comment:
They aren't in preview mode anymore.  Can probably update the link to the GA version.